src/plugin.ts

```diff
@@ -42,5 +42,8 @@ export async function startStopTask(inputs: PluginInputs, env: Env) {
 }
 
 function sanitizeMetadata(obj: LogReturn["metadata"]): string {
-  return JSON.stringify(obj, null, 2).replace(/</g, "&lt;").replace(/>/g, "&gt;").replace(/--/g, "&#45;&#45;");
+  return JSON.stringify(obj, null, 2)
+    .replace(/</g, "&lt;")
+    .replace(/>/g, "&gt;")
+    .replace(/--/g, "&#45;&#45;")
```

Comment:
Why specifically two dashes are escaped, but not single ones?

Reply:
One hyphen is grammatically correct while two is usually code-like isn't? I also removed the other escapes because it made the comment unreadable